packages/front-end/pages/settings/billing.tsx

```diff
@@ -105,6 +103,17 @@ const BillingPage: FC = () => {
           </p>
         )}
       </div>
+      {subscription?.status ? (
+        <>
+          <PaymentInfo />
+          {organization.id ? (
+            <div className="p-3 app-box border">
+              <h3>Invoices & Usage</h3>
+              <OrbPortal orgId={organization.id} />
+            </div>
+          ) : null}
+        </>
+      ) : null}
```

Comment:
This logic makes it so orgs with in-active orb subscriptions will see the portal - this way they can get old invoices if needed. This means it will also show usage (which will be empty). Open to feedback on this, as this could be confusing if an org saw it, but got a message that they're over their included usage in a free plan. 

The `/usage` route will display the correct Clickhouse-based usage data. Given that embedding the portal is an MVP and we'll eventually build our own direct integration, I think this is ok, but open to feedback.